usagov/vote-gov-drupal
# Bug/vote 1833 favicon path update

## Jira ticket

[Vote-1833](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1833&assignee=611272e0eb7aef00694b8a97)

## Description

This PR will update the config for the favicon to remove `[site:URL]` from the paths 
## Deployment and testing

### QA/Test

1. pull down branch 
2. start local 
3. run `lando retune` 
4. visit the following state pages and make sure there are no favicon errors in the console `Arkansas`, `American Somoa`, `Alaska`, `North Dakota`, and `Wyoming`

## Checklist for the Developer




## Checklist for the Peer Reviewers
